source/clear-linux/guides/maintenance/telemetry-enable.rst

```diff
@@ -42,12 +42,12 @@ To start telemetry on your system, run the following command:
 

```

Comment:
At lines 38 and 53, section title is not in sentence case ("Telemetry" in section title should be lower case).